src/main/java/pl/piekoszek/gorskimatches/repository/AccountRepository.java

```diff
@@ -0,0 +1,12 @@
+package pl.piekoszek.gorskimatches.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+import pl.piekoszek.gorskimatches.token.AccountInfo;
+
+@Component
```

Comment:
`@Repository` is enough, remove `@Component` ;)

Reply:
ok